src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -27,7 +27,7 @@
 
 class DeployerSanity:
     def __init__(self):
-        self.settings = Settings.settings
+        self.settings = Settings("/root/r7_custom.ini")
```

Comment:
Why do we have an internal rack name here?

Reply:
that was left by mistake. will correct it. Thanks

Reply:
Updated